### PR TITLE
translating workshops into pt-br

### PR DESCRIPTION
**Pull request by caioortiz27**

### CODE REVIEW

**Review by ozhang22 — COMMENTED** [review]
Left a few comments on files which are not yet translated.

I highly suggest getting Paulo and Joao to split reviewing the text for grammar and spelling.

**Comment by aminueza** [comment]
@caioortiz27 do you plan to continue translating to pt-br?

File: content/brazilian-portuguese/arduino/Appendix.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+description: ""
+draft: false
+weight: 8
+---
+
+# Apêndice
+
+## Os Componentes, em detalhe
+
+Imagine que o nosso circuito é uma peça de teatro. Antes da apresentação final diante de uma plateia, precisamos de um lugar para praticar. É aqui que entra a tábua de pão.
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/Appendix.md:12]: Fix "tábua de pão". Suggestion: "placa de ensaio"

File: content/brazilian-portuguese/arduino/Appendix.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+draft: false
+weight: 8
+---
+
+# Apêndice
+
+## Os Componentes, em detalhe
+
+Imagine que o nosso circuito é uma peça de teatro. Antes da apresentação final diante de uma plateia, precisamos de um lugar para praticar. É aqui que entra a tábua de pão. 
++ Breadboards são plataformas simples que podemos usar para construir circuitos temporários para teste e depuração antes de finalizar nosso projeto e soldá-lo permanentemente (soldar) juntos.
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/Appendix.md:13]: "soldá-lo", and then "(soldar)" is redundant

File: content/brazilian-portuguese/arduino/Appendix.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+# Apêndice
+
+## Os Componentes, em detalhe
+
+Imagine que o nosso circuito é uma peça de teatro. Antes da apresentação final diante de uma plateia, precisamos de um lugar para praticar. É aqui que entra a tábua de pão. 
++ Breadboards são plataformas simples que podemos usar para construir circuitos temporários para teste e depuração antes de finalizar nosso projeto e soldá-lo permanentemente (soldar) juntos. 
++ As placas de pão permitem-nos adicionar, remover ou reposicionar facilmente componentes. Abaixo, à esquerda, há uma imagem de uma tábua de pão.
++ A imagem à direita mostra como os pinos estão conectados uns aos outros. 
++ Cada uma das linhas numeradas na tábua de pão, mostrada a amarelo, contém 5 orifícios de alfinetes. Cada orifício de pino em uma linha (linha amarela) é conectado aos outros orifícios de pino nessa linha.
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/Appendix.md:16]: Fix "alfinetes". Suggestion: "pinos".

File: content/brazilian-portuguese/arduino/Appendix.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
++ Breadboards são plataformas simples que podemos usar para construir circuitos temporários para teste e depuração antes de finalizar nosso projeto e soldá-lo permanentemente (soldar) juntos. 
++ As placas de pão permitem-nos adicionar, remover ou reposicionar facilmente componentes. Abaixo, à esquerda, há uma imagem de uma tábua de pão.
++ A imagem à direita mostra como os pinos estão conectados uns aos outros. 
++ Cada uma das linhas numeradas na tábua de pão, mostrada a amarelo, contém 5 orifícios de alfinetes. Cada orifício de pino em uma linha (linha amarela) é conectado aos outros orifícios de pino nessa linha. 
++ A linha vermelha à direita mostra uma conexão comum entre esses orifícios de pino, enquanto a linha azul no lado direito mostra uma conexão comum entre esses orifícios de pino. 
++ O mesmo acontece nas linhas vermelha e azul à esquerda. No entanto, as linhas vermelha e azul à direita e à esquerda não estão conectadas entre si.
++ Componentes que estão conectados à mesma conexão comum são conectados no circuito
+
+![Alt Text: Imagem de uma tábua de pão sem linhas](../img/img16.png)
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/Appendix.md:21]: Fix "tábua de pão"

File: content/brazilian-portuguese/arduino/Appendix.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+![Alt Text: Imagem de um breadboad](../img/breadboard.png)
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/Appendix.md:22]: Translate "breadboard"

File: content/brazilian-portuguese/arduino/Appendix.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+O diodo emissor de luz (ou LED) é uma pequena luz que se acende quando a eletricidade flui através dele. 
+
++ O LED tem 2 derivações: o Ânodo e o cátodo.
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/Appendix.md:26]: The LED has 2 leads, which in this case, means "terminais", not "derivações"

File: content/brazilian-portuguese/arduino/Appendix.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
++ O ânodo é mais longo que o cátodo e está sempre ligado à tensão mais alta.
+![Alt Text: Imagem de um LED](../img/LED.png)
+
+O resistor é um pequeno pedaço de material cerâmico que retarda o fluxo de eletricidade. 
+
++ Um resistor tem 2 derivações, mas é omnidirecional. Em outras palavras, não importa qual chumbo você anexa à tensão mais alta.
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/Appendix.md:32]: Resistor has 2 leads, "terminais"

File: content/brazilian-portuguese/arduino/_index.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+difficulty: "Intermediário"
+icon: "fas fa-microchip"
+draft: false
+---
+
+## Introdução - Construindo com Arduino
+
+Eletrônicos como microcontroladores, sensores e todos os tipos de circuitos estão ao nosso redor! Alguns são muito básicos e fornecem informações de telemetria para sistemas de tráfego. Outros executam os semáforos programados com temporização embutida.
+
+Hoje trabalharemos em algumas atividades para mostrar algumas das capacidades desta plataforma. Começaremos pequenos e simples e construiremos para maiores e mais complexos. Você aprenderá todas as informações básicas sobre como usar a placa controladora Arduino, sensores e componentes. Depois de fazer cada atividade, sinta-se à vontade para mudar algumas coisas para ver o que é diferente e obter uma compreensão mais profunda
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/_index.md:14]: "mudar algumas coisas para ver o que é diferente" doesn't fit it well. Suggestion: "mudar alguns parâmetros para ver o que muda"

File: content/brazilian-portuguese/arduino/_index.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+![Picture of an Arduino](./img/Arduino.png)
+
+## Os Componentes
+
+ Os componentes que usaremos para o projeto são mostrados abaixo. Certifique-se de ter todos esses componentes antes de começar:
+
+![Alt Text: Picture of Elegoo UNO Starter Kit](img/Elegoo-starter-kit.png)
+
+`Elegoo` O Kit Super Iniciante de Projeto UNO com Tutorial para Arduino inclui todo o hardware (com exceção do computador)
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/_index.md:24]: "`Elegoo` O Kit Super Iniciante de Projeto UNO" is a product, I suggest to not translate it. Suggestion: "`Elegoo` UNO Project Super Starter Kit"

**Review by Renato3005 — CHANGES_REQUESTED** [review]
Appendix and Index reviewd

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+ biblioteca funciona com todos os monitores LCD que são compatíveis com o
+ Hitachi HD44780 driver. Há muitos deles por aí, e você
+ geralmente pode dizê-los pela interface de 16 pinos.
+
+Este esboço imprime "Hello World!" no LCD
+ e mostra o tempo.
+
+O circuito:
+ * Pino RS LCD para pino digital 7
+ * LCD Ativar pino para pino digital 8
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:31]: Fix "Ativar". Suggestion: "pino de ativação".

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+ * LCD D4 pino para pino digital 9
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:32]: Reorder the words. Suggestion: "Pino D4 do LCD para pino digital 9". Do the same in lines 33, to 38.

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+ * LCD D5 pino para pino digital 10
+ * LCD D6 pino para pino digital 11
+ * LCD D7 pino para pino digital 12
+ * LCD R / W pino ao solo
+ * LCD VSS pino à terra
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:37]: "à terra" isn't the correct meaning. Suggestion: "ao terra".

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+ e mostra o tempo.
+
+O circuito:
+ * Pino RS LCD para pino digital 7
+ * LCD Ativar pino para pino digital 8
+ * LCD D4 pino para pino digital 9
+ * LCD D5 pino para pino digital 10
+ * LCD D6 pino para pino digital 11
+ * LCD D7 pino para pino digital 12
+ * LCD R / W pino ao solo
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:36]: Fix "ao solo". Suggestion: "ao terra"

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+ * LCD VSS pino à terra
+ * LCD VCC pino para 5V
+ * Resistor de 10K:
+ * termina a +5V e terra
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:40]: Fix "e terra"

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+ * limpador para pino VO LCD (pino 3)
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:41]: add "VO **do** LCD". Confirm that steps on line 40 and 41 are understandable

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+
+Biblioteca adicionada originalmente 18 abr 2008
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:43]: add "originalmente **em** 18 abr 2008"

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 17]
```diff
@@ -0,0 +1,77 @@
...
+ por David A. Mellis
+ biblioteca modificada 5 jul 2009
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:45]: add "modificada **em** 5 jul"

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+ por Limor Fried (http://www.ladyada.net)
+ exemplo adicionado 9 jul 2009
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:47]: add "adicionado **em** 9 jul"

File: content/brazilian-portuguese/arduino/activity-4.md [hunk 19]
```diff
@@ -0,0 +1,77 @@
...
+ por Tom Igoe
+ modificado 22 nov 2010
```
Comment (Renato3005) [inline content/brazilian-portuguese/arduino/activity-4.md:49]: add "modificado **em** 22 nov"

**Review by Renato3005 — CHANGES_REQUESTED** [review]
All activities reviewd

**Comment by caioortiz27** [comment]
> @caioortiz27 do you plan to continue translating to pt-br?

@aminueza hello Amanda, yes - I would like to sync with you and Paulo on the next steps

**Pull request closed without merging**